packages/common/test/sites/feeds/setFeedTemplate.test.ts

```diff
@@ -1,30 +1,227 @@
-import * as legacyFeedsModule from "../../../src/sites/feed-configuration";
 import { setFeedTemplate } from "../../../src/sites/feeds/setFeedTemplate";
 import {
   IFeedsConfiguration,
   FeedFormat,
 } from "../../../src/sites/feeds/types";
 
 describe("setFeedTemplate", () => {
-  let setFeedConfigurationSpy: jasmine.Spy;
-  beforeEach(() => {
-    setFeedConfigurationSpy = spyOn(
-      legacyFeedsModule,
-      "setFeedConfiguration"
-    ).and.callThrough();
-  });
-  it("delegates to setFeedConfiguration", () => {
-    const feedsConfig: IFeedsConfiguration = {};
+  it("sets DCAT US V1 configuration", async () => {
+    const updatedTemplate = {
+      title: "{{new_title}}",
+    };
+    const feedsConfig: IFeedsConfiguration = {
+      dcatUS1X: {
+        title: "{{old_title}}",
+      },
+    };
     const format: FeedFormat = "dcat-us";
     const version = "1.1";
-    const updatedTemplate = { foo: "bar" };
     const result = setFeedTemplate({
       feedsConfig,
       format,
       version,
       updatedTemplate,
     });
-    expect(result).toEqual({ dcatUS1X: { foo: "bar" } });
-    expect(setFeedConfigurationSpy).toHaveBeenCalledTimes(1);
+    expect(result.dcatUS1X).toBeDefined();
+    expect(result.dcatUS1X).toEqual(updatedTemplate);
+  });
+  it("sets DCAT US V3 configuration", async () => {
+    const updatedTemplate = {
+      title: "{{new_title}}",
+    };
+    const feedsConfig: IFeedsConfiguration = {
+      dcatUS1X: {
```

Comment:
We should probably change this to `dcatUS3X:` to make sure that the old template is overwritten.

Reply:
Good catch! I have fixed it.